src/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -452,6 +464,53 @@ public async Task<ActionResult<DataPatchResponse>> PatchFormData(
         [FromBody] DataPatchRequest dataPatchRequest,
         [FromQuery] string? language = null
     )
+    {
+        var request = new DataPatchRequestMultiple()
+        {
+            Patches = new() { [dataGuid] = dataPatchRequest.Patch },
+            IgnoredValidators = dataPatchRequest.IgnoredValidators
+        };
+        var response = await PatchFormDataMultiple(org, app, instanceOwnerPartyId, instanceGuid, request, language);
+
+        if (response.Result is OkObjectResult { Value: DataPatchResponseMultiple newResponse })
```

Comment:
Is it possible to just check `response.Value` instead? The return from `PatchFormDataMultiple` might change to `StatusCode(200, ...)` for example which is also a valid `ActionResult<>` but then `.Result` is not this object. Feels kind of brittle